apps/gatsby/src/AppConfig/AppConfig.js

```diff
@@ -150,26 +157,79 @@ export class AppConfig extends React.Component {
     }
   };
 
-  toggleContentType = (enabledContentTypes, ctId) => {
-    if (enabledContentTypes.includes(ctId)) {
-      return enabledContentTypes.filter((cur) => cur !== ctId);
+  selectorTypeToggle = () => {
+    this.setState((prevState) => ({
+      selectorType: !prevState.selectorType
+    }))
+  }
+
+  disableContentType = (id) => {
+    const index = this.state.enabledContentTypes.findIndex(type => type === id)
```

Comment:
Maybe we can just `filter` and check on the output length?

Reply:
You mean check the output length of filtered vs. unfiltered array, and set state if the filtered array is less?

Reply:
I have meant (variables named very poorly, just to give an idea)

```
const newEnabledContentTypes = this.state.enabledContentTypes.filter(type => type !== id)
const hasToUpdate = this.state.enabledContentTypes > newEnabledContentTypes.length

if (hasToUpdate) {
  this.setState(() => ({ enabledContentTypes: newEnabledContentTypes }))
}
```

The idea is avoid skimming through the array twice. Not a big deal, since I don't expect this to be huge